test/centroid.cpp

```diff
@@ -0,0 +1,47 @@
+#include <catch.hpp>
+
+#include <math.h>
+#include <iostream>
+
+namespace lost {
+    int RowOrColumn(long i, int size, int subdivisions, int div, int leftover);
```

Comment:
Add a test for StartOfSubdivision to make sure it's actually the inverse of RowOrColumn. The test can just check that RowOrColumn(StartOfSubdivision(x)) == x for a whole bunch of different x values. (Might be useful to use the GENERATE macro to avoid retyping the same thing many times, see https://github.com/catchorg/Catch2/blob/devel/docs/generators.md or look at the geometry.cpp tests to see how to use it)